packages/fxa-settings/src/components/images/index.stories.tsx

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { Meta } from '@storybook/react';
+
+import {
+  RecoveryCodesImage,
+  MailImage,
+  HeartsBrokenImage,
+  HeartsVerifiedImage,
+  TwoFactorAuthImage,
+} from '.';
+import { withLocalization } from '../../../.storybook/decorators';
+
+export default {
+  title: 'Components/Images',
+  subcomponents: {
+    RecoveryCodesImage,
+    MailImage,
+    HeartsBrokenImage,
+    HeartsVerifiedImage,
+    TwoFactorAuthImage,
+  },
+  decorators: [withLocalization],
+} as Meta;
+
+export const RecoveryCodesIllustration = () => <RecoveryCodesImage />;
+
+export const MailIllustration = () => <MailImage />;
+
+export const HeartsBrokenIllustration = () => <HeartsBrokenImage />;
+
+export const HeartsVerifiedIllustration = () => <HeartsVerifiedImage />;
+
+export const TwoFactorAuthIllustration = () => <TwoFactorAuthImage />;
```

Comment:
nit: I noticed that we refer to these images as "images" (when components), "illustrations" (when exports/stories), and "graphics" (when filenames).

Not sure if there is a way to standardize on one, but just thought I'd mention it. Probably not super relevant since most people will just import a single image directly or multiples via packages/fxa-settings/src/components/images/index.tsx.

Reply:
Yeah I've been mulling over this one also. I think they're most accurately illustrations...? but I also don't want to change the existing labels that call them "graphics" 🤔

Reply:
Personally I like `image` everywhere since that's what we called the directory name that houses them all but 🤷‍♀️